src/Components/LifecycleChart/LifecycleChart.tsx

```diff
@@ -119,16 +120,16 @@ const LifecycleChart: React.FC<LifecycleChartProps> = ({ lifecycleData }: Lifecy
     }
     if (dataType === 'appLifecycle') {
       (lifecycleData as Stream[]).forEach((item) => {
-        if (item.start_date === 'Unknown' || item.end_date === 'Unknown') {
+        if (item.start_date === 'Unknown' || item.end_date === 'Unknown' || item.os_major === 8 || item.start_date === null || item.end_date === null) {
```

Comment:
We want to filter 8 out at the top level rather than individually at the chart level - it also impacts the table.